e2e/handles/handles.test.ts

```diff
@@ -1,19 +1,26 @@
 //  Handles test suite
 import "@frequency-chain/api-augment";
 import assert from "assert";
-import { createDelegator, hasRelayChain } from "../scaffolding/helpers";
+import { createDelegator } from "../scaffolding/helpers";
 import { KeyringPair } from "@polkadot/keyring/types";
 import { MessageSourceId } from "@frequency-chain/api-augment/interfaces";
 import { ExtrinsicHelper } from "../scaffolding/extrinsicHelpers";
 import { Bytes } from "@polkadot/types";
 import { getBlockNumber } from "../scaffolding/helpers";
+import { hasRelayChain } from "../scaffolding/env";
+import { getFundingSource } from "../scaffolding/funding";
 
 describe("🤝 Handles", () => {
+    const fundingSource = getFundingSource("handles");
+
     let msa_id: MessageSourceId;
     let msaOwnerKeys: KeyringPair;
+
+    let expirationOffset = hasRelayChain() ? 10 : 100;
```

Comment:
When running in instant sealing mode, this needs to be LARGER due to how fast everything is going.